rclcpp/minimal_subscriber/wait_set.cpp

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021, Apex.AI Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <iostream>
+#include <memory>
+
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/string.hpp"
+
+/* This example creates a subclass of Node and uses a wait-set based loop to wait and handle
+ * messages from the subscriber */
+
+class MinimalSubscriber : public rclcpp::Node
```

Comment:
Similar request, let's call it `WaitSetSubscriber` and change the file name to match.

Reply:
renamed